### PR TITLE
Add workflow for performance tests in CI

### PR DESCRIPTION
**Pull request by ArtDu**
Now we can run these tests on a trigger in UI or via github rest.
Fixed unstoppable BenchmarkRunner.
Also added a manuall trigger in the default workflow.

You can see the result here https://github.com/tarantool/cartridge-java/actions/runs/2280786760

I didn't forget about

- [x] Tests (It's ci)
- [x] Changelog 
- [x] Documentation (it's iternal changes)
- [x] Cleanup the code for review. See [checklist](https://github.com/tarantool/cartridge-java/blob/master/docs/review-checklist.md)

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # This workflow will build a Java project with Maven
 # For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
 
-name: ubuntu-master
+name: Tests
 
 on:
+  workflow_dispatch:
```
Comment (dkasimovskiy) [inline .github/workflows/tests.yml:7]: Why do we need to run tests manually?

**Review by dkasimovskiy — CHANGES_REQUESTED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (ArtDu) [reply .github/workflows/tests.yml:7]: It's not necessary, but i thought why not? 😁

Comment (dkasimovskiy) [reply .github/workflows/tests.yml:7]: If it's not nessasary we no need it here.

Comment (ArtDu) [reply .github/workflows/tests.yml]: Removed

**Review by dkasimovskiy — APPROVED** [review]

**Pull request merged**